src/main/java/com/metamx/common/RetryUtils.java

```diff
@@ -23,6 +23,7 @@
 
 import java.util.Random;
```

Comment:
This is the reason why I want to introduce checkstyle.. https://github.com/druid-io/druid/pull/3551